src/test/java/de/doubleslash/usb_led_matrix/SettingsTest.java

```diff
@@ -1,112 +1,43 @@
 package de.doubleslash.usb_led_matrix;
 
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import java.io.File;
+import java.io.IOException;
 
-class SettingsTest {
-   @BeforeEach
-   void resetSettings() {
-      Settings.reset();
-   }
-
-   @Test
-   void shouldSetBrightnessValueWhenConfiguredWithLongCommandLineOption() {
-      String[] args = { "--brightness", "60" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getBrightness()).isEqualTo(60);
-   }
-
-   @Test
-   void shouldSetBrightnessValueWhenConfiguredWithShortCommandLineOption() {
-      String[] args = { "-b", "60" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getBrightness()).isEqualTo(60);
-   }
-
-   @Test
-   void shouldSetModeToManualWhenConfiguredWithLongCommandLineOption() {
-      String[] args = { "--mode", "manual" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getMode()).isEqualTo("manual");
-   }
-
-   @Test
-   void shouldSetModeToManualWhenConfiguredWithShortCommandLineOption() {
-      String[] args = { "-m", "manual" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getMode()).isEqualTo("manual");
-   }
-
-   @Test
-   void shouldSetColorModeToDarkWhenConfiguredWithLongCommandLineOption() {
-      String[] args = { "--colorMode", "dark" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getColorMode()).isEqualTo("dark");
-   }
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 
-   @Test
-   void shouldSetColorModeToDarkWhenConfiguredWithShortCommandLineOption() {
-      String[] args = { "-c", "dark" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getColorMode()).isEqualTo("dark");
-   }
-
-   @Test
-   void shouldSetTimeoutWhenConfiguredWithLongCommandLineOption() {
-      String[] args = { "--timeout", "30" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getTimeout()).isEqualTo(30);
-   }
-
-   @Test
-   void shouldSetTimeoutWhenConfiguredWithShortCommandLineOption() {
-      String[] args = { "-t", "30" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getTimeout()).isEqualTo(30);
-   }
-
-   @Test
-   void shouldSetComPortWhenConfiguredWithLongCommandLineOption() {
-      String[] args = { "--port", "COM7" };
-      Settings.handleParameters(args);
-      assertThat(Settings.getCom()).isEqualTo("COM7");
-   }
+class SettingsTest {
```

Comment:
As we discussed in person we are still missing a test for the working functionality of loading the colors from the settings file.

Reply:
Done.